app/src/pages/prompt/PromptChatMessages.tsx

```diff
@@ -97,3 +99,17 @@ function ChatMessages({
     </Flex>
   );
 }
+
+export function PromptChatMessagesCard({
+  title = "Prompt",
+  promptVersion,
+}: {
+  title?: string;
+  promptVersion: PromptChatMessages__main$key;
+}) {
+  return (
+    <Card title={title} variant="compact">
+      <PromptChatMessages promptVersion={promptVersion} />
+    </Card>
+  );
+}
```

Comment:
super minor nit but if this is the main component to be consumed as a unit - I would probably just re-name the file

Reply:
Ah yes I meant to do that, good catch

Reply:
Done